package.json

```diff
@@ -105,9 +105,13 @@
     "gulp": "^4.0.2",
     "gulp-filter": "^5.1.0",
     "gulp-gzip": "^1.4.2",
+    "gulp-inject": "^5.0.5",
     "gulp-jsbeautifier": "^2.1.2",
     "gulp-jshint": "^2.0.0",
     "gulp-load-plugins": "^1.5.0",
```

Comment:
Same deal here, as this version has some vulnerability and they fix it on 1.6.0, but still, the only major difference here set a minimum Node version. WDYT?

Reply:
Same as above, final changes to package.json are only about adding needed dependencies. Existing versions are unchanged.

Reply:
No problem, it's your call.